centreon/src/Core/Security/Vault/Application/UseCase/UpdateVaultConfiguration/UpdateVaultConfiguration.php

```diff
@@ -35,6 +35,7 @@
     NotFoundResponse,
     PresenterInterface
 };
+use Core\Common\Application\Repository\ReadVaultRepositoryInterface;
```

Comment:
I wonder if we shouldn't rename the useCase in UpsertVaultConfiguration instead of UpdateVaultConfiguration

Reply:
I thought about but there is another endpoint named Update that is in fact an upsert so I kept the same logic